lib/web_ui/test/canvaskit/common.dart

```diff
@@ -23,18 +23,19 @@ void setUpCanvasKitTest() {
   );
 
   tearDown(() {
-    HtmlViewEmbedder.instance.debugClear();
-    RenderCanvasFactory.instance.debugClear();
+    CanvasKitRenderer.instance.debugClear();
   });
```

Comment:
Thorough teardown in the `canvaskit` common :P

Reply:
:)